content/cards/attendee/test-users.md

```diff
@@ -0,0 +1,10 @@
+###Testing with Users
+Great! You've created something for your project!
+
+Now we want to get user feedback to make sure our assumptions match customer expectations. Hopefully by now, you have a relationship with a partner or partners to which you can throw your newly-developed software. 
+
+Be sure to put the product in front of them to how they use it, and incorporate feedback into your next iteration. Validate that you're solving the right problem, and check to make sure you've reduced any friction involved in the user experience. Pivot as necessary.
+
+This cycle of refining after demoing should go a long way towards injecting life into the project. That should translate into the customer getting a lot of value from our efforts!
+
+For more information on UX research, see [this presentation](https://docs.google.com/presentation/d/1GKp98UVZXyxo2fjeOfV87TbJb8Z4zF7MozZ4ILtOO_k/edit)  given at one of our Code for Denver Meetups.
```

Comment:
We should probably verify that Michael Fleetwood is comfortable with us posting this openly online first.